packages/core/src/prismic/index.ts

```diff
@@ -1,10 +1,11 @@
-import { startServerAndOpenBrowser } from "./auth";
-import { Poll, Endpoints } from "../utils";
-import type { Manifest, Frameworks, UserInfo } from "../models";
+import type { Manifest, Frameworks } from "../models";
 import * as Communication from "./communication";
-import { PrismicSharedConfigManager } from "../filesystem/PrismicSharedConfig";
 import { Repositories } from "../models/Repositories";
+
 export * as Communication from "./communication";
+export * as Endpoints from "./endpoints";
+export * from "./SharedConfig";
+
 export interface Core {
```

Comment:
I wondering, do we still need to export this, is it used ?
Does it makes sense that it's in this folder called 'Prismic' ?

Reply:
the core export?

Reply:
let's find out :)

Reply:
yes